Allow resetting the VimuxCloseOnExit option

mvanderkamp opened the pull request:
This was an oversight on my part. Anyone using the autoclose option would be unable to prevent the runner from closing when vim exits by simply doing `:let g:VimuxCloseOnExit = v:false`, since the option was only considered when the plugin script is sourced, so the value when vim closes was ignored. This PR remedies that situation.

m42e commented:
@mvanderkamp  Could you rename the function to match the common `Vimux<FunctionName>` stype

mvanderkamp commented:
> @mvanderkamp Could you rename the function to match the common `Vimux<FunctionName>` stype

Should that style apply to script-local functions? I could move it next to the other script-local functions.

m42e commented:
You are right, not all functions follow that convention. 
Ok, move it there and make it somewhat like `isAutocloseActive` or somthing like that if you agree.

mvanderkamp commented:
K I've moved the function, and I renamed the `augroup` to be more general in case we find reasons to add more `autocmd`s later. I left the function name as-is though since I think it is sufficiently precise already.
